libultraship/libultraship/Lib/Fast3D/U64/PR/ultra64/gbi.h

```diff
@@ -3129,12 +3129,12 @@ _DW({                                   \
 #define	gsDPSetCombineMode(a, b)	gsDPSetCombineLERP(a, b)
 #endif
 
-#if _MSC_VER
+#if defined(_MSC_VER) || defined(__GNUC__)
 #define CALL_2(A,B) A B
 #define CALL_3(A,B,C) A B C
 
-#define	gsDPSetCombineMode(a, b)	CALL_2(gsDPSetCombineLERP, (a, b))
-  //#define gsDPSetCombineMode(a, b)    _SHIFTL(0, 24, 8), 0
+// #define	gsDPSetCombineMode(a, b)	CALL_2(gsDPSetCombineLERP, (a, b))
```

Comment:
Why was this commented out?

Reply:
Again, MSVC's preprocessor not being strict enough with macro redefinition.